grails-app/conf/templates/configuration/users.csv

```diff
@@ -1,5 +1,5 @@
 username,firstName,lastName,email,roleType
 browser,Mr,Browser,,"ROLE_BROWSER,ROLE_FINANCE"
-purchaser,Ms,Purchaser,,"ROLE_ADMIN,ROLE_APPROVER,ROLE_FINANCE"
-accountant,Mr,Accountant,,"ROLE_ADMIN,ROLE_APPROVER,ROLE_FINANCE,ROLE_INVOICE"
-superuser,Ms,Superuser,,"ROLE_SUPERUSER,ROLE_APPROVER,ROLE_FINANCE,ROLE_INVOICE"
+purchaser,Ms,Purchaser,,"ROLE_ADMIN,ROLE_FINANCE"
```

Comment:
Is this because there's no role in the database with roletypecode = ROLE_APPROVER?

Reply:
That's right